### PR TITLE
Bulk update

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Bundle/test/templates/subscription/grid/bulk_action/accept.html.twig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <button type="submit">
+        <i class="icon trash"></i> Bulk accept
+    </button>
+    <input type="hidden" name="_csrf_token" value="{{ csrf_token('bulk_accept') }}" />
+
+    {% for resource in grid.data %}
+        <input type="hidden" name="ids[]" value="{{ resource.id }}" />
+    {% endfor %}
+</form>
+
```
Comment (Zales0123) [inline src/Bundle/test/templates/subscription/grid/bulk_action/accept.html.twig]: Extra blank line :dancer:

File: tests/Application/templates/grid/bulk_action/delete.html.twig [hunk 2]
```diff
@@ -11,3 +11,4 @@
         <input type="hidden" name="ids[]" value="{{ resource.id }}" />
     {% endfor %}
 </form>
+
```
Comment (Zales0123) [inline tests/Application/templates/grid/bulk_action/delete.html.twig]: And here :dancer:

File: tests/Application/src/Tests/Controller/SubscriptionUiTest.php [hunk 3]
```diff
@@ -177,6 +177,23 @@ public function it_allows_accepting_a_subscription(): void
...
+        $this->client->request('GET', '/admin/subscriptions');
+        $this->client->submitForm('Bulk accept');
+
+        $this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
+
+        /** @var Subscription $subscription */
+        $subscription = static::getContainer()->get('app.repository.subscription')->findOneBy(['email' => 'marty.mcfly@bttf.com']);
+
+        $this->assertNotNull($subscription);
+        $this->assertSame('accepted', $subscription->getState());
```
Comment (Zales0123) [inline tests/Application/src/Tests/Controller/SubscriptionUiTest.php]: As this is "bulk" update we should probable check in the test state of multiple subscriptions to be sure it works correctly

Comment (Zales0123) [reply tests/Application/src/Tests/Controller/SubscriptionUiTest.php]: Also, can we somehow test the scenario where only some of existing subscriptions are bulk accepted (e.g. have 3 subscriptions in fixtures and accept only 2 of them)?

Comment (loic425) [reply tests/Application/src/Tests/Controller/SubscriptionUiTest.php]: done for the two of them.

Comment (loic425) [reply tests/Application/src/Tests/Controller/SubscriptionUiTest.php]: ah god, I failed with your second one. I've added one of them that is already accepted. That's not simple cause it was a lot easier to bulk all the items. Could you convert this to an issue on the board? We already know that it works cause I use this dev branch for my slides tests. But we need to refactor these tests and templates for that.

**Review by Zales0123 — APPROVED** [review]

**Comment by Zales0123** [comment]
Thanks, Loïc! :tada:

**Pull request merged**